crates/re_types/definitions/rerun/blueprint/archetypes/space_view_blueprint.fbs

```diff
@@ -0,0 +1,40 @@
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+
+namespace rerun.blueprint.archetypes;
+
+// ---
+
+/// The top-level description of the Viewport.
+table SpaceViewBlueprint (
+    "attr.docs.unreleased",
+    "attr.rerun.scope": "blueprint",
```

Comment:
the more I see this the more I wish this would be enforced by being in the blueprint folder. Don't know how to do that nicely though yet. Tbd